frontend/packages/kubevirt-plugin/src/plugin.ts

```diff
@@ -48,6 +50,18 @@ const plugin: Plugin<ConsumedExtensions> = [
       mergeAfter: 'Pods',
     },
   },
+  {
+    type: 'NavItem/ResourceNS',
+    properties: {
+      section: 'Workloads',
+      componentProps: {
+        name: 'Virtual Machine Templates',
+        resource: models.VmTemplateModel.plural,
```

Comment:
This shouldn't be a resource page since we have an existing resource page for templates (and not all templates are VM templates). This will need to be a special route.

Reply:
Thanks, :+1: will address this in #1673 :-)